x-pack/plugins/lens/public/datasources/form_based/operations/definitions/metrics.tsx

```diff
@@ -318,7 +318,7 @@ export const averageOperation = buildMetricOperation<AvgIndexPatternColumn>({
   quickFunctionDocumentation: i18n.translate(
     'xpack.lens.indexPattern.avg.quickFunctionDescription',
     {
-      defaultMessage: 'The average value of a number field.',
+      defaultMessage: 'The mean value of a set of number fields.',
```

Comment:
While technically the same, why the decision to change from `average` to `mean`?

Reply:
This was based off feedback we received from Sophie.

Reply:
I agree with @KOTungseth on this one - for some countries average can be interpreted as 'mean average' or 'median average' so changing it to mean makes sense :)